README.md

```diff
@@ -27,25 +28,38 @@
 ## General
 
 - [FRC Penultimate Manual](https://github.com/MC42/FRCPenultimateManual) - All-in-one manual for FRC teams and team members.
-- [The Blue Alliance](https://github.com/the-blue-alliance/the-blue-alliance) - FRC's leading event, team, and match tracker.
+- [The Blue Alliance](https://www.thebluealliance.com/) - FRC's leading event, team, and match tracker.
+- [The Compass Alliance](https://www.thecompassalliance.org/) - A resource for learning about all FRC topic, made for any level of experience.
 - [State Space Guide](https://github.com/calcmogul/state-space-guide) - A practical guide to state-space control: graduate-level control theory for high schoolers.
 - [GradleRIO](https://github.com/wpilibsuite/GradleRIO) - A powerful Gradle Plugin that allows teams competing in the FIRST robotics competition to produce and build their code.
 
+## Suppliers
```

Comment:
Add newline after

Reply:
Thanks! Did you see anything else that needed to be fixed or changed?